File: 1132 - Multiplos de 13/multiplos_de_13.py
import sys

dados = sys.stdin.read().split()
x, y = int(dados[0]), int(dados[1])
if x > y:
    x, y = y, x
soma = 0
for i in range(x, y + 1):
    if i % 13 != 0:
        soma += i
print(soma)

File: 1132 - Multiplos de 13/multiplos_de_13.c
#include <stdio.h>

int main(){
    int x, y;
    scanf("%d %d", &x, &y);
    if(x > y){ int t = x; x = y; y = t; }
    int soma = 0;
    for(int i = x; i <= y; i++)
        if(i % 13 != 0) soma += i;
    printf("%d\n", soma);
    return 0;
}
